trace/tracestate.go

```diff
@@ -121,7 +120,7 @@ func ParseTraceState(tracestate string) (TraceState, error) {
 		}
 
 		if _, ok := found[m.Key]; ok {
-			return TraceState{}, wrapErr(errDuplicate)
+			continue // Duplicate is ignored per https://github.com/w3c/trace-context/blob/551a5b0855171281e98b4c2a814bf9e1f837cd53/test/test.py#L563-L568.
```

Comment:
We can also choose to overwrite instead of ignoring.
This would also comply with the specification. It would simplify the code and we could get rid of `found` map.